editor/inserter/style.scss

```diff
@@ -137,6 +138,14 @@ input[type="search"].editor-inserter__search {
 	width: 100%;
 
 	&.is-active {
-		border-bottom: 2px solid #f00;
+		background: $white;
+		border-bottom: 2px solid $blue-medium-500;
```

Comment:
Not a big deal, but this is cleaner if it were just border-bottom-color: $blue-medium-500.